DEVELOPING_OPAL/validate/src/test/scala/org/opalj/fpcf/PurityTests.scala

```diff
@@ -97,4 +102,29 @@ class PurityTests extends PropertiesTest {
         validateProperties(as, contextsWithAnnotations(as.project), Set("Purity"))
     }
 
-}
+    describe(
+        "the org.opalj.fpcf.analyses.L2PurityAnalysis is executed "+
+            "together with the L0FieldImmutabilityAnalysis and L2FieldAssignabilityAnalysis"
```

Comment:
Is there enough of a conceptual difference between L1 and L2 field assignability? If the latter essentially subsumes the first, we should probably only test with that one?